src/PostgREST/Plan.hs

```diff
@@ -680,28 +692,32 @@ generateSpreadSelectFields ReadPlan{select, relSelect} =
 --
 -- The second tuple contains the aggregate function to be applied, the cast, and
 -- the alias, if it was supplied by the user or otherwise determined.
+--
+-- No hoisting is done for to-many spreads, since it makes sense to
+-- apply the aggregates at the level of their respective subqueries.
```

Comment:
I wonder if this "hoisting" could be mentioned somewhere in https://docs.postgrest.org/en/v12/references/api/aggregate_functions.html. I tend to forget why we do this.

Reply:
I added it in #3913